brew_view/specification.py

```diff
@@ -226,7 +226,7 @@
             },
             "socket_timeout": {
                 "type": "int",
-                "default": 3000,
+                "default": 13000,
```

Comment:
Should also add the `previous_defaults` field to this so it gets upgraded during a migration.